Source/common/SNTCommonEnums.h

```diff
@@ -36,13 +36,20 @@ typedef NS_ENUM(NSInteger, SNTAction) {
 #define RESPONSE_VALID(x) \
   (x == SNTActionRespondAllow || x == SNTActionRespondDeny || x == SNTActionRespondAllowCompiler)
 
+// Supported Rule Types
+//
+// IMPORTANT: These enum values should be in order of decreasing precedence as
```

Comment:
Why?

Reply:
Agree with @russellhancox here not sure why this matters since we're always taking the first rule we get out of the sqlite database so the identifiers could be anything as long as the order in the query is right. This seems like an unnecessary callout.

Reply:
The goal of the PR is to protect against what is (I thought) an undocumented side effect of sqlite evaluation, described here: https://github.com/google/santa/blob/main/Source/santad/DataLayer/SNTRuleTable.m#L336

I would argue that even if this is a documented and guaranteed feature of sqlite, we are still better served by having the rule types be in order of precedence as it makes things more readable.

Definitely happy to comment.